examples/src/multi-file/api.ts

```diff
@@ -1,3 +1,4 @@
 import "./endpoints/create-user";
 import "./endpoints/delete-user";
+import "./endpoints/get-users";
```

Comment:
How about `find-users` to avoid confusion with `get-user`?

Reply:
changed